dataprofiler/profilers/graph_profiler.py

```diff
@@ -299,10 +309,27 @@ def _get_continuous_distribution(self, graph, continuous_attributes):
                     if mle <= best_mle:
                         best_fit = distribution.name
                         best_mle = mle
+
+                properties = []
+
+                mean, variance, skew, kurtosis = distribution.stats(fit, moments="mvsk")
+
+                if best_fit == "lognorm":
+                    shape, loc, scale = fit
+                    properties = [shape, loc, scale, mean, variance, skew, kurtosis]
+                elif best_fit == "gamma":
+                    a, loc, scale = fit
+                    properties = [a, loc, scale, mean, variance, skew, kurtosis]
+                else:
+                    loc, scale = fit
+                    properties = [loc, scale, mean, variance, skew, kurtosis]
+
```

Comment:
we can clean this up a bit ... we know that `[loc, scale, mean, variance, skew, kurtosis]` will always be in the list... so the if / else should really just add `a` or `shape` to the list .... else `if not lognorm or gamma` just keep moving and have propertyes == `[loc, scale, mean, variance, skew, kurtosis]`